modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java

```diff
@@ -43,6 +43,24 @@
  *
  */
 public class ConsistencyCommand extends AbstractCommand<Object> {
+    /** Cache. */
+    public static final String C = "-c";
+
+    /** Cache. */
+    public static final String CACHE = "--cache";
```

Comment:
Other commands uses enums for list of params. Let's do it in the same way.

Reply:
At least some of the commands are strings:
ClusterStateChangeCommand#FORCE_COMMAND
DefragmentationCommand#NODES_ARG
WalCommands#WAL_PRINT
...

Reply:
OK, but it's sad :(